app/containers/CreateTag/Form.jsx

```diff
@@ -38,49 +39,64 @@ export const Form = ({
   handleSubmit,
   translations,
   communities,
-}) => (
-  <FormBox onSubmit={handleSubmit(createTag)}>
-    <Field
-      className={isSingleCommunityWebsite() ? 'd-none' : ''}
-      name={FORM_COMMUNITY}
-      component={CommunityField}
-      disabled={createTagLoading}
-      label={translations[messages.community.id]}
-      tip={translations[messages.communityTip.id]}
-      options={communities}
-      validate={[requiredForObjectField]}
-      warn={[requiredForObjectField]}
-      splitInHalf
-    />
+  suggestedTags,
+  getSuggestedTagsDispatch,
+}) => {
+  const onChange = value => {
+    if (value) {
+      getSuggestedTagsDispatch({ communityId: value.id })
+    }
+  };
 
-    <Field
-      disabled={createTagLoading}
-      name={NAME_FIELD}
-      component={TextInputField}
-      label={translations[messages.name.id]}
-      tip={translations[messages.nameTip.id]}
-      validate={[strLength2x15, required, valueHasNotBeInList]}
-      warn={[strLength2x15, required, valueHasNotBeInList]}
-      splitInHalf
-    />
+  return (
+    <FormBox onSubmit={handleSubmit(createTag)}>
+      <Field
+        className={isSingleCommunityWebsite() ? 'd-none' : ''}
+        name={FORM_COMMUNITY}
+        component={CommunityField}
+        disabled={createTagLoading}
+        label={translations[messages.community.id]}
+        tip={translations[messages.communityTip.id]}
+        options={communities}
+        validate={[requiredForObjectField]}
+        warn={[requiredForObjectField]}
+        splitInHalf
+        onChange={onChange}
+      />
 
-    <Field
-      disabled={createTagLoading}
-      name={DESCRIPTION_FIELD}
-      component={TextareaField}
-      label={translations[messages.descriptionField.id]}
-      tip={translations[messages.descriptionFieldTip.id]}
-      validate={[strLength20x1000, required]}
-      warn={[strLength20x1000, required]}
-      splitInHalf
-    />
+      <Field
+        disabled={createTagLoading}
+        name={NAME_FIELD}
+        component={TextInputField}
+        label={translations[messages.name.id]}
+        tip={translations[messages.nameTip.id]}
+        validate={[
+          strLength2x15,
+          required,
+          valueHasNotBeInList,
```

Comment:
No need to add additional validator. Need to add suggested tag to the list of values for `valueHasNotBeInList`. 
See line in this file: 
```
valueHasNotBeInListValidate: (
state?.toJS()?.form?.[FORM_NAME]?.values?.[FORM_COMMUNITY]?.tags ?? []
).map(x => x.name)
```
Suggested tags need to be added to that list.

Reply:
But we can get aggested tags by asynchronous action and this value isn't exist in time of creating 'valueHasNotBeInListValidate'. Do I have to do asynchronous validator?